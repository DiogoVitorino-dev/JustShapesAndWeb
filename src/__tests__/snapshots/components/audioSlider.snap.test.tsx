import { render } from "@testing-library/react-native";

import AudioSlider from "@/components/settings/audio/audioSlider";

describe("Audio Slider - Snapshot component test", () => {
  it("should render correctly", () => {
    const tree = render(<AudioSlider />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
