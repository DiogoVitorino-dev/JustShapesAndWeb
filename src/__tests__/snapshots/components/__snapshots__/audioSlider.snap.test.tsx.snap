// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Audio Slider - Snapshot component test should render correctly 1`] = `
<View
  style={
    [
      undefined,
      [
        {
          "minWidth": 90,
          "overflow": "hidden",
        },
        undefined,
      ],
    ]
  }
>
  <Image
    collapsable={false}
    forwardedRef={[Function]}
    jestAnimatedStyle={
      {
        "value": {},
      }
    }
    source={1}
    style={
      {
        "height": "100%",
        "overflow": "hidden",
        "position": "absolute",
        "width": "100%",
      }
    }
  />
  <Image
    collapsable={false}
    jestAnimatedStyle={
      {
        "value": {},
      }
    }
    source={1}
    style={
      {
        "height": "100%",
        "overflow": "hidden",
        "position": "absolute",
        "width": "100%",
      }
    }
  />
  <View
    onLayout={[Function]}
    style={
      [
        {
          "defaultIndicatorIdle": {
            "backgroundColor": "#C0C0C0",
            "height": 10,
            "width": 2,
          },
          "defaultIndicatorMarked": {
            "backgroundColor": "#CCCCCC",
            "height": 20,
            "width": 5,
          },
          "defaultSlider": {},
          "defaultSlideriOS": {
            "height": 40,
          },
          "sliderMainContainer": {
            "width": "100%",
            "zIndex": 1,
          },
          "stepIndicatorElement": {
            "alignContent": "center",
            "alignItems": "center",
          },
          "stepNumber": {
            "alignItems": "center",
            "marginTop": 20,
            "position": "absolute",
          },
          "stepsIndicator": {
            "flex": 1,
            "flexDirection": "row",
            "justifyContent": "space-between",
            "top": 10,
            "zIndex": 2,
          },
          "thumbImage": {
            "alignContent": "center",
            "alignItems": "center",
            "position": "absolute",
          },
          "thumbImageContainer": {
            "alignContent": "center",
            "alignItems": "center",
            "justifyContent": "center",
            "position": "absolute",
            "zIndex": 3,
          },
          "trackMarkContainer": {
            "alignContent": "center",
            "alignItems": "center",
            "alignSelf": "center",
            "justifyContent": "center",
            "position": "absolute",
            "zIndex": 3,
          },
        },
        [
          [
            {
              "zIndex": 10,
            },
          ],
          {
            "height": 40,
          },
        ],
        {
          "justifyContent": "center",
        },
      ]
    }
  >
    <RNCSlider
      disabled={false}
      inverted={false}
      lowerLimit={-9007199254740991}
      maximumTrackTintColor="transparent"
      maximumValue={100}
      minimumTrackTintColor="transparent"
      minimumValue={0}
      onChange={[Function]}
      onRNCSliderAccessibilityAction={null}
      onRNCSliderSlidingComplete={null}
      onRNCSliderSlidingStart={null}
      onRNCSliderValueChange={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      step={0}
      style={
        [
          {
            "width": 0,
            "zIndex": 1,
          },
          {
            "height": 40,
          },
          {
            "alignContent": "center",
            "alignItems": "center",
          },
        ]
      }
      tapToSeek={true}
      thumbTintColor="#00000000"
      upperLimit={9007199254740991}
    />
  </View>
</View>
`;
